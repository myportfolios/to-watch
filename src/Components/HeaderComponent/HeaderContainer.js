import React, { Component } from "react";
import HeaderPresentation from "./HeaderPresentation";

export default class HeaderContainer extends Component {
  render() {
    return <HeaderPresentation />;
  }
}
