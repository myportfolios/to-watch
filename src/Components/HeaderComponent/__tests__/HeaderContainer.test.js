import { shallow } from "enzyme";
import HeaderContainer from "../HeaderContainer";
import React from "react";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("headerContainer", () => {
  let wrapper = shallow(<HeaderContainer />);

  it("renders correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
