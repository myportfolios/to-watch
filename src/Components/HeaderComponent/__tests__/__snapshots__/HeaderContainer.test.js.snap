// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`headerContainer renders correctly 1`] = `ShallowWrapper {}`;
